app/models/concerns/exportable.rb

```diff
@@ -36,15 +36,66 @@ module Exportable
     "Clinic" => :export_clinic_name,
     "Pledge sent" => :pledge_sent,
     "Resolved without fund assistance" => :resolved_without_fund,
-    "Pledge generated time" => :pledge_generated_at
+    "Pledge generated time" => :pledge_generated_at,
+
+    "Timestamp of first call" => :first_call_timestamp,
+    "Timestamp of last call" => :last_call_timestamp,
+    "Call count" => :call_count,
+    "Reached Patient call count" => :reached_patient_call_count,
+
+    "Fulfilled" => :fulfilled,
+    "Procedure date" => :procedure_date,
+    "Gestation at Procedure" => :gestation_at_procedure,
```

Comment:
We'll want to clarify that this is in weeks. Right now it's returned as an int which isn't terribly clear. I'll fix it up.